docs/guides/testing-dbus.rst

```diff
@@ -0,0 +1,290 @@
+Testing D-Bus Services
+======================
+Class :class:`sssd_test_framework.utils.sbus.SBUSUtils` provides an API to
+operate on D-Bus services. This class has two methods:
+
+* :meth:`sssd_test_framework.utils.sbus.SBUSUtils.getInfoPipe()`
+* :meth:`sssd_test_framework.utils.sbus.SBUSUtils.getDestination()`
+
+It is important to notice that you don't need to instantiate an object of class
+:class:`sssd_test_framework.utils.sbus.SBUSUtils` because the ``Client`` role already
+provides one when you invoke the
+:meth:`sssd_test_framework.roles.client.Client.sbus()` method.
+
+:meth:`sssd_test_framework.utils.sbus.SBUSUtils.getInfoPipe()` returns a preinstantiated
+object of class :class:`sssd_test_framework.utils.sbus.DBUSDestination`, which gives you
+direct access to the `Infopipe` destination; while the method
+:meth:`sssd_test_framework.utils.sbus.SBUSUtils.getDestination()` allows you to
+create a new instance of :class:`sssd_test_framework.utils.sbus.DBUSDestination`
+for which you can specify the actual `destination` (a server) and the ``bus`` to
+use to contact that destination.
+
+.. code-block:: python
+    :caption: Getting a destination
+
+    ifp_dest = client.sbus().getInfoPipe()
+    monitor_dest = client.sbus().getDestination(dest="sssd.monitor", bus=DBUSKnownBus.MONITOR)
+
+
+The bus to provide to :meth:`sssd_test_framework.utils.sbus.SBUSUtils.getDestination()`
+must be an object of class :class:`sssd_test_framework.utils.sbus.DBUSKnownBus` which
+provides some predefined buses:
+
+* :attr:`sssd_test_framework.utils.sbus.DBUSKnownBus.SYSTEM`
+* :attr:`sssd_test_framework.utils.sbus.DBUSKnownBus.SESSION`
+* :attr:`sssd_test_framework.utils.sbus.DBUSKnownBus.MONITOR`
+
+(If no bus is provided, the system bus is used.)
```

Comment:
I'd rephrase this and add the following note: `The default bus is system.`

Reply:
Fixed (although not pushed yet)

Reply:
By note I meant something like `.. note::` as it's being used in other parts of the document

Reply:
Done